assets/js/components/notifications/FirstPartyModeSetupBanner.stories.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * FirstPartyModeSetupBanner Component Stories.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import fetchMock from 'fetch-mock';
+
+/**
+ * Internal dependencies
+ */
+import { provideModules } from '../../../../tests/js/utils';
+import WithRegistrySetup from '../../../../tests/js/WithRegistrySetup';
+import { CORE_USER } from '../../googlesitekit/datastore/user/constants';
+import { withNotificationComponentProps } from '../../googlesitekit/notifications/util/component-props';
+import { WEEK_IN_SECONDS } from '../../util';
+import FirstPartyModeSetupBanner from './FirstPartyModeSetupBanner';
+import {
+	CORE_NOTIFICATIONS,
+	NOTIFICATION_AREAS,
+} from '../../googlesitekit/notifications/datastore/constants';
+import { VIEW_CONTEXT_MAIN_DASHBOARD } from '../../googlesitekit/constants';
+
+const FPM_BANNER_ID = 'first-party-mode-setup-cta-banner';
+
+const NotificationWithComponentProps = withNotificationComponentProps(
+	FPM_BANNER_ID
+)( FirstPartyModeSetupBanner );
+
+function Template() {
+	return <NotificationWithComponentProps />;
+}
+
+export const Default = Template.bind();
+Default.storyName = 'FirstPartyModeSetupBanner';
+Default.scenario = {
+	label: 'Modules/FirstPartyMode/Dashboard/FirstPartyModeSetupBanner',
+};
```

Comment:
The `label` can be omitted, it will be automatically generated from the `title` and `storyName` properties. We are aiming to remove unnecessary use of the `label` property.

```suggestion
Default.scenario = {}:
```